app/models/casa_case.rb

```diff
@@ -20,7 +21,7 @@ class CasaCase < ApplicationRecord
   TRANSITION_AGE_YOUTH_ICON = "🦋".freeze
   NON_TRANSITION_AGE_YOUTH_ICON = "🐛".freeze
 
-  before_create :set_slug
+  friendly_id :case_number, :use => :scoped, :scope => :casa_org
```

Comment:
Colon syntax preferred over hash rocket style:

```suggestion
  friendly_id :case_number, use: :scoped, scope: :casa_org
```

Reply:
Thanks! Looks like this was picked up by the linter too, which is good. :)